.github/workflows/release.yml

```diff
@@ -12,19 +12,15 @@ jobs:
 
   release:
     needs: prepare
-    name: Release
+    name: publish
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Setup Node.js
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
         with:
           node-version: 14
-      - name: Install dependencies
-        run: npm install
-      - name: Release
+          registry-url: https://registry.npmjs.org/
+      - run: npm install
+      - run: npm publish
```

Comment:
I think you'll have to manually bump the version in the package.json for this to publish to a new version (and not fail). Since this is a test, I'd just do a patch version bump.